upgrades/3.x/3.12.0/README.adoc

```diff
@@ -27,9 +27,17 @@ Starting in version 3.12.0, APIKeys now contain distinct attributes:
 
 The 3 Portal API endpoints listed below now expose distinct *id* and *key* in the HTTP response (previously, the key attribute was exposed as '*id*') :
 
-- GET /subscriptions/{subscriptionId}
-- POST /subscriptions/{subscriptionId}
-- POST /subscriptions/{subscriptionId}/_renew
+- GET /subscriptions/{subscription.id}
+- POST /subscriptions/{subscription.id}
+- POST /subscriptions/{subscription.id}/_renew
+
+=== Deprecated Endpoints
+
+The 3 Rest API endpoints listed below are now deprecated, and will be removed in a future version.
+
+- PUT /apis/{api.id}/keys/{apiKey.key} : Replaced by PUT /keys/{apiKey.id}
+- DELETE /apis/{api.id}/keys/{apiKey.key} : Replaced by DELETE /keys/{apiKey.id}
+- POST /apis/{api.id}/keys/_verify?apiKey={apiKey.key} : Replaced by GET /keys/_canCreate?api={api.id}&application={application.id}&key={apiKey.key}
```

Comment:
maybe use a table ?

Reply:
yes, changed it to a table, it looks better like that. Thanks